src/sentry/integrations/opsgenie/integration.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+# import logging
```

Comment:
Can we delete unused imports instead of commenting them out? We can always add them back later.